pipeline/psi-only-executor.groovy

```diff
@@ -51,6 +51,14 @@ node("rhel-8-medium || ceph-qe-ci") {
 
         // Trigger psi_only pipeline based on ciMessage content
         if (ciMap.pipeline.final_stage && ciMap.pipeline.tags.contains("tier-0") && ciMap.test.result == "SUCCESS") {
+            def recipeFileContent = sharedLib.yamlToMap("${rhcephVersion}.yaml")
+            def content = recipeFileContent['latest']
+            println "recipeFile ceph-version : ${content['ceph-version']}"
+            println "buildArtifacts ceph-version : ${ciMap.recipe.'ceph-version'}"
+            if ( ciMap.recipe."ceph-version" != content['ceph-version']) {
```

Comment:
This may not abort the stages if psi-only execution already started for any stage.
We need to add more checks in tier executor to abort psi-only execution.

Reply:
Made required changes in tier executor to abort psi-only execution.